extra/Documentation/eng/TechInfo.ChangeTracking.md

```diff
@@ -0,0 +1,118 @@
+This temporary file keeps track of the changes to `eng\TechInfo.txt`. Please review the initial change list.
```

Comment:
Do we really need this file in the repository if it will be removed eventually?
Maybe keep it as an issue?

Reply:
No, we do not. The intent was to simply reviewing / commenting the original version (which worked just fine). I created Issue #174 for ongoing tracking (and even not sure we need that).